app/models/article.rb

```diff
@@ -1,8 +1,8 @@
 class Article < ApplicationRecord
-	validates :title, presence: true,
-									length: { minimum: 5 }
-	
-	validates :body, presence: true
+  validates :title, presence: true,
+  length: { minimum: 5 }
```

Comment:
minor style suggestion:
I think this will fit on one line just fine. `validates :title, presence: true, length: { minimum: 5 }`